wremnants/muon_calibration.py

```diff
@@ -149,6 +178,71 @@ def get_dummy_uncertainties():
 
     return h
 
+def make_jpsi_crctn_helper(filepath):
+    f = uproot.open(filepath)
+
+    # TODO: convert variable axis to regular if the bin width is uniform
+    A, e, M = [x.to_hist() for x in [f['A'], f['e'], f['M']]]
+
+    # TODO: make this into a function in utilities/boostHistHelpers
+    if (A.axes != e.axes) or (e.axes != M.axes):
+        raise RuntimeError("A, e, M histograms have different axes!")
+    else:
+        axis_param = hist.axis.Regular(3, 0, 3, underflow = False, overflow = False, name = 'param')
+        axis_cenval = hist.axis.Regular(1, 0, 1, name = 'cenval')
```

Comment:
Is this dimension just for consistency with the uncertainty hist?

Reply:
yes, it's just that. I was writing the uncertainty helper by inheriting from the central value helper, but later found that it would be too cumbersome.

Reply:
Wait, so the extra dimension isn't used now? If not I would remove it, it's needlessly confusing

Reply:
okay I got rid of it. Let's wait for the CI tests to run through to make sure I didn't break up anything.